stages/install_bootstrap/main.py

```diff
@@ -77,26 +79,37 @@ def main():
     else:
         log("command", cmd, out.splitlines())
 
-    # Download and install bear
+    # Build and install bear
     with tempfile.TemporaryDirectory() as d:
-        url = ("https://github.com/karkhaz/Bear/blob/master/"
-               "bear-2.1.5-1-x86_64.pkg.tar.xz?raw=true")
-        response = urllib.request.urlopen(url)
-        tar_file = response.read()
-        pkg_name = "bear.pkg.tar.xz"
-        with open(os.path.join(d, pkg_name), "wb") as f:
-            f.write(tar_file)
+        bear_tar = os.path.join(d, "bear.tar.xz")
```

Comment:
It is created on the next two lines (it's not something that exists before the stage runs). The next two lines create a tarball of the bear source, which locally mounted to this stage.